engine/admin/resource_constraints.md

```diff
@@ -38,16 +38,22 @@ for [Memory Resource Controller](https://www.kernel.org/doc/Documentation/cgroup
 
 ### `--memory-swap` details
 
-- If unset, and `--memory` is set, the container can use twice as much swap
+- A limit value equal to memory plus swap. Must be used with `-m (--memory)` flag.
+      The limit should always be larger than `-m (--memory)` value. By default, it is
+      set to double the value of --memory.
```

Comment:
Yes, one of those statements must be wrong I agree. The kernel docs do not mention the double memory thing https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt but it may be that Docker does that I need to check the code (its DockerCon and I won't have time today!)

I remember looking at this before and finding it confusing, I think there have been a few issues about it previously, and we should resolve it.